internal/cmd/org.go

```diff
@@ -22,6 +23,9 @@ func init() {
 	membersCmd.AddCommand(membersAddCmd)
 	membersCmd.AddCommand(membersRemoveCmd)
 	membersCmd.AddCommand(membersInviteCmd)
+	membersCmd.AddCommand(inviteRemoveCmd)
+	membersCmd.AddCommand(inviteListCmd)
```

Comment:
I suggest we group those commands under `turso org invites`
Then you'd have:

- `turso org invites list`
- `turso org invites create`
- `turso org invites remove`

`turso org invite` (or `turso org members invite`) could be an alias to `turso org invites create`, if you like the idea - not asking you to do it though.